wtf/graph_widget.go

```diff
@@ -0,0 +1,160 @@
+package wtf
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/rivo/tview"
+)
+
+type GraphWidget struct {
```

Comment:
I think this should be renamed to not include `Widget`. In WTF a directory of executable files that comprise a single service are a "module", and the part that is effectively `main` that manages the content that will be displayed onscreen, is called a "widget".  Might be confusing (for instance, above, `bargraph/widget/Widget` is including `GraphWidget`).

How about calling this just `BarGraph`?

Reply:
sounds good.